cg/meta/workflow/nallo.py

```diff
@@ -18,3 +22,72 @@ def __init__(
         workflow: Workflow = Workflow.NALLO,
     ):
         super().__init__(config=config, workflow=workflow)
+        self.root_dir: str = config.nallo.root
+        self.workflow_bin_path: str = config.nallo.workflow_bin_path
+        self.profile: str = config.nallo.profile
+        self.conda_env: str = config.nallo.conda_env
+        self.conda_binary: str = config.nallo.conda_binary
+        self.platform: str = config.nallo.platform
+        self.params: str = config.nallo.params
+        self.workflow_config_path: str = config.nallo.config
+        self.resources: str = config.nallo.resources
+        self.tower_binary_path: str = config.tower_binary_path
+        self.tower_workflow: str = config.nallo.tower_workflow
+        self.account: str = config.nallo.slurm.account
+        self.email: str = config.nallo.slurm.mail_user
+        self.compute_env_base: str = config.nallo.compute_env
+        self.revision: str = config.nallo.revision
+        self.nextflow_binary_path: str = config.nallo.binary_path
+
+    @property
+    def sample_sheet_headers(self) -> list[str]:
+        """Headers for sample sheet."""
+        return NalloSampleSheetHeaders.list()
+
+    def get_sample_sheet_content_per_sample(self, case_sample: CaseSample) -> list[list[str]]:
+        """Collect and format information required to build a sample sheet for a single sample."""
+        bam_unmapped_read_paths = self.get_unmapped_bam_read_paths(sample=case_sample.sample)
+        sample_sheet_entries = []
+
+        for bam_path in bam_unmapped_read_paths:
+            sample_sheet_entry = NalloSampleSheetEntry(
+                project=case_sample.case.internal_id,
+                sample=case_sample.sample.internal_id,
+                bam_unmapped_read_path=Path(bam_path),
+                family_id=case_sample.case.internal_id,
+                paternal_id=case_sample.get_paternal_sample_id or "0",
+                maternal_id=case_sample.get_maternal_sample_id or "0",
+                sex=self.get_sex_code(case_sample.sample.sex),
+                phenotype=self.get_phenotype_code(case_sample.status),
+            )
+            sample_sheet_entries.extend(sample_sheet_entry.reformat_sample_content)
+        return sample_sheet_entries
```

Comment:
In this context can, can we call it `read_paths` or `read_files`? I feel like unmapped_bam is a detail that can remain in the method that fetches it, but is not relevant in the wider context

Reply:
sounds good 👍 changed to read_file, and the method that fetches it to get_bam_read_file_paths.